src/components/Tab/Tab.spec.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { renderWithTheme } from '../../../test/utils';
+import Tab from './Tab';
+
+describe('<Tab />', () => {
+  describe('prop: children', () => {
+    it('should render passed child', () => {
+      const child = 'Hey there!';
+      const { getByText } = renderWithTheme(<Tab>{child}</Tab>);
+
+      expect(getByText(child)).toBeInTheDocument();
+    });
+  });
+
+  describe('prop: selected', () => {
+    it('should render with correct aria attribute', () => {
+      const { getByRole } = renderWithTheme(<Tab selected />);
+
+      const tab = getByRole('tab');
+      expect(tab).toHaveAttribute('aria-selected', 'true');
```

Comment:
Since this tab variable is only being used inside the `expect` why don't you do something like:

```expect(getByRole('tab')).toHaveAttribute('aria-selected', 'true');```

?

Reply:
fixed by c471290